typescript/src/openapi-client/cdpApiClient.ts

```diff
@@ -144,7 +144,7 @@ export const cdpApiClient = async <T>(
           "network_connection_failed",
           "Unable to connect to CDP service. The service may be unavailable.",
           { code: error.code, message: error.message, retryable: true },
-          error.cause,
+          error.cause instanceof Error ? error.cause : undefined,
```

Comment:
It may be better to update our type to be more permissive and accept `unknown`. If an "unknown" error were to occur, we wouldn't be capturing it